symbols/overlay13.yml

```diff
@@ -39,7 +39,7 @@ overlay13:
         NA: 0x238A1C8
         JP: 0x238B728
       description: |-
-        Note: unverified, ported from Irdkwia's notes
+        Handles the 'return' value from MENU_PERSONALITY_TEST called by scripts. 
```

Comment:
I am confident that the function handles the return values of the menu because it calls FUN_022e6e8 (which has a debug print for menu result) with values that line up with the values used in the quiz script from the diagram I made below.
https://discord.com/channels/710190644152369162/1134395025724293211/1134395025724293211

The other switch function handles the progression of the quiz itself from one step to the next.

Admittedly, my knowledge about the scripting engine is lacking so I don't feel confident in giving them a name without understanding how the functions are called/used within the scripting engine. It's possible that the function is MENU_PERSONALITY_TEST itself, but I don't know enough to say so confidently.

Reply:
Okay, we'll leave it as is then.